client/src/modules/list/components/Items/MoveToListPanel/MoveToListPanel.jsx

```diff
@@ -0,0 +1,207 @@
+import React, { Fragment, PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router-dom';
+import _flowRight from 'lodash/flowRight';
+import { FormattedMessage, injectIntl } from 'react-intl';
+
+import Filter from 'common/components/Filter';
+import { fetchListsForItem } from 'modules/list/model/actions';
```

Comment:
1)Please change the name, I would go for 'fetchListsForUser'
beacuse I can see in the controller that you are fetching lists for userId, not for itemId.
2) Can't you just use "fetchListsMetaData" logic?? Instead of creating a second controllers, actions etc ??

Reply:
Ad 2) I tried to use fetchListMetaData logic but I realized that I have to get lists in which the user is at least a member, not viewer because only as a member I can manage list's items